src/main/java/Control/Scenario/UC_CreateFlat.java

```diff
@@ -0,0 +1,211 @@
+package Control.Scenario;
+
+import Control.Connect.DbProvider;
+import Model.Database.Interaction.*;
+import Model.Database.Support.Assurance;
+import Model.Database.Tables.Table.*;
+import Model.Web.JsonResponse;
+import Model.Web.Specific.FlatFirstTimeCreation;
+import Model.Web.Specific.FlatOwner;
+import View.Support.CustomExceptions.CreationException;
+import org.apache.commons.validator.routines.EmailValidator;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.constraints.NotNull;
+import java.sql.SQLException;
+import java.util.Dictionary;
+import java.util.Hashtable;
+import java.util.List;
+
+import static Model.Database.Support.DbConfig.DB_DO_NOT_USE_THIS_FILTER;
+
+public class UC_CreateFlat {
+    private DbProvider db;
+
+    public UC_CreateFlat(@NotNull DbProvider dbProvider) {
+        this.db = dbProvider;
+    }
+
+
+    public final @NotNull JsonResponse createNewFlat_Owner_Controller(@NotNull final FlatFirstTimeCreation firstTimeCreation) {
```

Comment:
preco FlatFirstTimeCreation? Chces to odlisit ked sa bude zapisovat ten flat pod inym majitelom?

Reply:
toto bol velky problem nejako spravit totiz, nevies pridat flat bez toho aby si mala nejaky controller, preto potrebujes spravit tieto veci naraz